### PR TITLE
Upgraded osmpbfreader 0.5.0 to 0.6.3

### PR DESCRIPTION
**Pull request by datanel**

### CODE REVIEW

File: src/boundaries.rs [hunk 1]
```diff
@@ -11,7 +11,7 @@
 // LICENCE: This program is free software; you can redistribute it
 // and/or modify it under the terms of the GNU Affero General Public
 // License as published by the Free Software Foundation, either
-// version 3 of the License, or (at your option) any later version.
+// version 3 of the License, or (at your option) any decimicro_later version.
```
Comment (TeXitoi) [inline src/boundaries.rs]: oups...

**Comment by TeXitoi** [comment]
Ha, maybe another change, now you have a method `OsmPbfReader::get_objs_and_deps`, thus maybe changing the call from the free function to the method as I will deprecated the free function in the next major release.

**Comment by datanel** [comment]
@TeXitoi done

**Comment by TeXitoi** [comment]
:+1:

**Pull request merged**